scripts/docs/generate-configuration-vocabulary-crosswalk.py

```
@@ -0,0 +1,227 @@
+    "type": "PayloadToolDecision (assay-evidence)",
+    "fields": "policy_digest, policy_snapshot_digest (+_alg/_canonicalization/_schema), "
+    "tool_definition_digest (+_alg/_canonicalization/_schema/_source), args_schema_hash",
+    "note": "All optional, no doc comments, and populated by no committed fixture. Semantics are "
+    "stated nowhere and there is no instance to read them from, so **no relation to any field "
+    "above can be asserted**. `policy_digest` reads like a shorter `declared_policy_digest`; that "
+    "resemblance is a name, not evidence.",
+}
+
+
+def run(root: Path, *args: str) -> str:
+    return subprocess.run(["git", "-C", str(root), *args],
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)